Pool/My Project/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
See if we can remove this as this branch is targeted towards more GUI. If it's needed then save it for branch 1.8.3 next.